rhino/src/main/java/org/mozilla/javascript/JavaMembers.java

```diff
@@ -328,7 +328,8 @@ private void discoverAccessibleMethods(
                                 if (isPublic(mods) || isProtected(mods) || includePrivate) {
                                     MethodSignature sig = new MethodSignature(method);
                                     if (!map.containsKey(sig)) {
-                                        if (includePrivate) method.trySetAccessible();
+                                        if (includePrivate && !method.isAccessible())
+                                            method.setAccessible(true);
```

Comment:
I stumbled upon this place in #1575 (which probably will cause merge conflicts)
I'm wondering, if we should replace this with `VMBridge.instance.tryToMakeAccessible(method)`
which will effectively call this code: https://github.com/mozilla/rhino/blob/master/rhino/src/main/java/org/mozilla/javascript/jdk18/VMBridge_jdk18.java#L57

Reply:
Maybe that's a good idea. I guess it will work on Android too.